react-locate-contractor/tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "noImplicitAny": false
```

Comment:
this should probably be true

Reply:
If your getting errors on this, its more indicative of bad code than a bad setting. This is a linter setting meant to protect you from having implicit dynamics which is bad code practice

Reply:
Makes sense. I was able to get it fixed.